hackIDE/templates/hackIDE/index.html

```diff
@@ -77,6 +78,11 @@
 								<span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
 							</a>
```

Comment:
You don't need to add any new element in HTML. Just keep everything same in HTML and replace the previous download function calling with your own one.